system-ui-showcases/src/index.stories.tsx

```diff
@@ -1,37 +1,37 @@
 import React from 'react';
-import theme from './primer-theme';
 import { SystemUIShowcases } from './index';
 import { Space } from '~/space';
+import { defaultTheme } from './default-theme';
 
-export const space = () => <Space scale={theme.space} />;
+export const space = () => <Space scale={defaultTheme.space} />;
 
 export const width = () => (
   <SystemUIShowcases
-    theme={theme}
+    theme={{}}
     showcaseKey="width"
     componentProps={{
       height: '8rem',
       boxShadow: 'large',
-      bg: 'green.6'
+      bg: 'green.6',
     }}
   />
 );
 
 export const height = () => (
   <SystemUIShowcases
-    theme={theme}
+    theme={{}}
```

Comment:
can't it be ommited ?

Reply:
I wanted to leave it in the stories as an example of usage, but I will remove it.